Tests/AsyncHTTPClientTests/SOCKSEventsHandlerTests+XCTest.swift

```diff
@@ -0,0 +1,35 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the AsyncHTTPClient open source project
+//
+// Copyright (c) 2018-2019 Apple Inc. and the AsyncHTTPClient project authors
```

Comment:
```suggestion
// Copyright (c) 2021 Apple Inc. and the AsyncHTTPClient project authors
```

Reply:
This is an autogenerated file, and so the header shouldn't be changed.